src/lib/commit.ts

```diff
@@ -81,24 +81,30 @@ export function formatGroupTime(timestamp: number): string {
 }
 
 export function groupCommits(
-  commits: { header: CommitHeader; context: CommitContext }[],
+  commits: { commit: CommitHeader; context: CommitContext }[],
 ): CommitGroup[] {
   const groupedCommits: CommitGroup[] = [];
   let groupDate: Date | undefined = undefined;
 
+  if (!window.HEARTWOOD) {
```

Comment:
We should do all of these transforms closer to where the API request is made, like we did in the first PR.
So in `src/lib/project.ts` the `getCommits` and `getCommit` functions. So that they're all in one place.